### ISSUE TITLE
BiglyBT and I2P only?

### ISSUE THREAD

**Issue by eurobank** [BiglySoftware/BiglyBT#3120]
Hi,

Trying to use Biglybt with ONLY I2P everywhere, i get a tracker error. And nothing else.

"Network not enabled for url 'udp://tracker ...................'

Is there some guide or something i could follow?

**Comment by parg** [comment]
What are you trying to achieve? I2P only connects to I2P peers. It doesn't have "exit nodes" like Tor does. So if you select I2P only then you can only connect to I2P peers using either the I2P DHT or explicit I2P trackers (domains ending in .i2p)

udp:// and http(s):// trackers only work over the public internet, which you have disable, hence "network not enabled"

**Comment by eurobank** [comment]
Ok, maybe i got all this wrong. I'm trying to get a *nix release iso from torrent.

**Comment by eurobank** [comment]
I gave this an other try. 

Using I2P torrent client "snark" (Windows 11) i'm able to get the metadata of a torrent file that doesn't have i2p tracker inside and the download starts,

I removed the standalone i2p and i try to do the same with Biglybt. Same file. And i'm stuck in metadata download. Tried others also, they do not start.

Probably i'm missing something very obvious here.

Any ideas? thanks
